src/api/course/index.ts

```diff
@@ -137,10 +137,84 @@ export class CourseApi {
           this.processV1DestinationDeltas(msg)
         }
       )
+      ;(this.app as any).subscriptionmanager.subscribe(
```

Comment:
I'm trying to remove all the `as` casts  everywhere, especially to `any`. Any chance you could set `app: ServerApp` above?

Reply:
subscriptionmanager is not part of ServerApp, it is an attribute of `app`

https://github.com/SignalK/signalk-server/blob/1785734d9438a490782dccf00664f7e00d455feb/src/index.ts#L324

Reply:
You are right, sorry about that. It will be part of `ServerAPI` after #1918 is merged.

Reply:
Let's try to merge #1918 first and adjust here.

Reply:
Updated post #1918 merge.